### ISSUE TITLE
CBSsports homepage elements aren't displayed properly

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/firefox-tv#2596]
## Steps to reproduce
1. Navigate to cbssports.com
2. Check the homepage elements

### Expected behavior
The homepage elements should be displayed in a certain way, as on Silk browser and Desktop.

### Actual behavior
The homepage elements are displayed in a different manner.

### Device information
* Fire TV device: Fire TV Stick 2nd Gen
* Latest affected Firefox version: **Firefox TV v4.1 LAT2**
* Earliest affected Firefox version: **Firefox TV v4.0**
* Turbo Mode: ON

### Notes
<details>
    <summary>Firefox for FireTV VS Silk & Desktop</summary>

► Firefox for FireTV
![FFTV](https://user-images.githubusercontent.com/51314259/61932790-eb54db80-af8c-11e9-847f-26d541287e68.jpg)

► Silk
![Silk](https://user-images.githubusercontent.com/51314259/61932802-ef80f900-af8c-11e9-87b0-43e1d718176e.jpg)

► Desktop 
![Desk](https://user-images.githubusercontent.com/51314259/61932820-fad42480-af8c-11e9-87b6-0283e9f151fe.jpg)

</details>

**Comment by athomasmoz** [comment]
@abodea I tried this out too, it's definitely a funky layout. From what I could tell, you're still able to navigate around and use the menu, so I wouldn't consider it a release-blocker. Would you agree?

**Comment by abodea** [comment]
@athomasmoz yes I agree but the layout somehow is really strange as the top bar somehow is below a video.
